File: app/src/main/java/app/lawnchair/lawnicons/screens/AboutFragment.java
package app.lawnchair.lawnicons.screens;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import app.lawnchair.lawnicons.ContributorsRecViewAdapter;
import app.lawnchair.lawnicons.Data;
import app.lawnchair.lawnicons.R;

public class AboutFragment extends Fragment {

    public AboutFragment(){

    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {

        View view = inflater.inflate(R.layout.fragment_about, container, false);
        RecyclerView contributorRecView = view.findViewById(R.id.conRecView);
        ContributorsRecViewAdapter adapter = new ContributorsRecViewAdapter(view.getContext());

        Data.setContext(view.getContext());
        adapter.setContributors(Data.contributors());
        contributorRecView.setAdapter(adapter);
        contributorRecView.setLayoutManager(new LinearLayoutManager(view.getContext()));

        return view;
    }
}

File: app/src/main/java/app/lawnchair/lawnicons/screens/about/AboutFragment.kt
package app.lawnchair.lawnicons.screens.about

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.recyclerview.widget.LinearLayoutManager
import app.lawnchair.lawnicons.Data
import app.lawnchair.lawnicons.databinding.AboutFragmentBinding
import com.google.android.material.transition.MaterialSharedAxis

class AboutFragment : Fragment() {

    private lateinit var binding: AboutFragmentBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        enterTransition = MaterialSharedAxis(MaterialSharedAxis.X, false)
        exitTransition = MaterialSharedAxis(MaterialSharedAxis.X, false)
    }

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        binding = AboutFragmentBinding.inflate(inflater, container, false)

        val contributorRecView = binding.conRecView
        val adapter = ContributorsRecViewAdapter(context)
        adapter.setContributors(Data.getContributors())
        contributorRecView.adapter = adapter
        contributorRecView.layoutManager =
            LinearLayoutManager(context)

        return binding.root
    }

}